docs/modals.html

```diff
@@ -121,17 +124,19 @@ <h1 class="wrapper-heading">Modals - CTA at the centre</h1>
                 </div>
 
                 <div class="component-showcase-col modals">
-                    <div class="modal-bg-content"></div>
-                    <div class="modal">
-                        <h1>Order Placed!</h1>
-                        <hr />
-                        <p class="p-lg">Congratulations! Your order has been confirmed.</p>
-                        <p class="p-lg">Please check your email for details.</p>
-                        <div>
-                            <button class="modal-cta icon-light">OK</button>
-                            <button class="modal-cta icon-dark"> Order Summary</button>
+                    <div class="modal-bg-content-centre">
+                        <div class="modal">
+                            <h1>Order Placed!</h1>
```

Comment:
Suggestion : dont use class names like `"modal"` you will get into trouble with CSS clashes when you use this component in react as common class names will override the CSS, use custom names like `"clay-modal"`

Reply:
Thank you, I didn't think of this. Made the changes.